README.md

```diff
@@ -317,6 +317,57 @@ curl -X 'PATCH' \
 }'
 ```
 
+#### adding database level instructions
+
+You can add database level instructions to the context store manually from the `POST /api/v1/instructions` endpoint
+These instructions are passed directly to the engine and can be used to steer the engine to generate SQL that is more in line with your business logic.
+
+```
+curl -X 'POST' \
+  '<host>/api/v1/instructions' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "instruction": "This is a database level instruction"
+  "db_connection_id": "db_connection_id"
+}'
+```
+
+#### getting database level instructions
+
+You can get database level instructions from the `GET /api/v1/instructions` endpoint
+
+```
+curl -X 'GET' \
+  '<host>/api/v1/instructions?page=1&limit=10' \
```

Comment:
you can set  `db_connection_id` here '<host>/api/v1/instructions?page=1&limit=10&db_connection_id=12312312'